scripts/verify-spelling.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########
+# This script verifies mispellings in location. Today it only supports
+# verifying English locale but can be modified in a future to support 
+# also other locales.
+# You need to run this script inside the root directory of "website" git repo.
```

Comment:
Previously, I found that the script **must** be run from the `website/scripts` directory, not from `website`.
Is the comment on line 20 correct?